### PR TITLE
@grafana/e2e: support regex for addDataSource

### PR DESCRIPTION
**Pull request by stevenvachon**
See commit message(s).

### CODE REVIEW

**Review by briangann — APPROVED** [review]
excellent!

**Review by ryantxu — APPROVED** [review]
LGTM

File: packages/grafana-e2e/src/flows/addDataSource.ts [hunk 1]
```diff
@@ -32,7 +32,14 @@ export const addDataSource = (config?: Partial<AddDataSourceConfig>): string =>
   form();
   e2e.pages.DataSource.saveAndTest().click();
   e2e.pages.DataSource.alert().should('exist');
-  e2e.pages.DataSource.alertMessage().should('contain.text', expectedAlertMessage);
```
Comment (hugohaggmark) [inline packages/grafana-e2e/src/flows/addDataSource.ts:35]: @stevenvachon did this not work https://github.com/cypress-io/cypress/issues/3747#issuecomment-605457187

Comment (stevenvachon) [reply packages/grafana-e2e/src/flows/addDataSource.ts:35]: It does, actually. I think that it's not obvious that it's an assertion, however.

Comment (stevenvachon) [reply packages/grafana-e2e/src/flows/addDataSource.ts:35]: Went with it, and added a comment.

**Pull request merged**